vite.config.ts

```diff
@@ -0,0 +1,10 @@
+import { resolve } from "path";
+import { defineConfig } from "vite";
+import dts from "vite-plugin-dts";
+import solid from "vite-plugin-solid";
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [solid(), dts()],
+  root: resolve("./dev"),
+});
```

Comment:
Can we move this file inside the `dev` folder. This will help keep things cleaner in the root directory